starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,80 @@
 // Names and Input
-
+Abi Summers Github abisummers
+Manon Salaun Github MSLN8
```

Comment:
Be careful to always comment out any expression that is not part of your code by adding // before it, otherwise it will throw errors when you try to execute your file. 
There are different shortcuts to comment multiple lines on mac and linux, try to find them it will be very useful :) (ctrl+K,ctrl+C works on any type of file on linux!)